.drone.star

```diff
@@ -2089,8 +2090,22 @@ def ocisServer(storage, accounts_hash_difficulty = 4, volumes = [], depends_on =
         "%s/bin/ociswrapper serve --bin %s --url %s --admin-username admin --admin-password admin" % (dirs["ocisWrapper"], ocis_bin, environment["OCIS_URL"]),
     ]
 
+    ocis = {
+        "name": container_name,
+        "image": OC_CI_GOLANG,
+        "detach": True,
+        "environment": environment,
+        "user": user,
+        "commands": [
+            "%s init --insecure true" % ocis_bin,
+            "cat $OCIS_CONFIG_DIR/ocis.yaml",
+        ] + wrapper_commands,
+        "volumes": volumes,
+        "depends_on": depends_on,
+    }
+
     wait_for_ocis = {
-        "name": "wait-for-%s" % (container_name),
+        "name": "wait-for-%s" % (ocis["name"]),
```

Comment:
you can keep this as it is

Reply:
Yes, I can. but I think this way is better.